securing_apps/topics/cli-apps/cli-apps.adoc

```diff
@@ -0,0 +1,137 @@
+[[_kcinit]]
+
+== Securing Command Line Applications
+
+Many applications need a command line interface to administer and configure themselves.
```

Comment:
Command line interface with what?
Configure themselves for what?

Perhaps something like this, with you adjusting my words for accuracy:
Many applications need a command line interface with an identity provider for the application to administer and configure itself.

Reply:
Maybe just:

"Many applications have a command line interface."

Reply:
That works and is simpler. I like it.